File: sdkwork-app-sdk-csharp/Models/CategoryCreateForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class CategoryCreateForm
    {
        public string? Name { get; set; }
        public string? Type { get; set; }
        public string? Code { get; set; }
        public string? Icon { get; set; }
        public string? Description { get; set; }
        public int? ParentId { get; set; }
        public int? Sort { get; set; }
        public int? Status { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/category_create_form.go
package types

// 创建分类请求
type CategoryCreateForm struct {
	Name string `json:"name"`
	Type string `json:"type"`
	Code string `json:"code"`
	Icon string `json:"icon"`
	Description string `json:"description"`
	ParentId int `json:"parentId"`
	Sort int `json:"sort"`
	Status int `json:"status"`
}
